.github/workflows/test-integration.yml

```
@@ -1,40 +1,70 @@
-name: test-integration
+name: Test Solidity Verifier
```

```suggestion
name: test-integration
```

Let's revert this this workflow isn't just for the solidity verifiers.